ChainRouter.php

```diff
@@ -306,4 +306,25 @@ public function getRouteCollection()
 
         return $this->routeCollection;
     }
+    
+    /**
+     * Identify if any routers have been added
+     * 
+     * @return bool
+     */
+    public function hasRouters()
+    {
+        return !empty($this->routers);
+    }
+    
+    /**
+     * Get the number of added routers
+     * 
+     * @return integer
+     */
+    public function getNumRouters()
+    {
+        return count($this->routers);
```

Comment:
do we really need this? i would suggest you do `count($chainRouter->all())` for this. the count would only ever be relevant when debugging, and there performance should not be critical. and the sorting gets cached, so when the chainrouter is also used in the request, $chainRouter->all() costs quasi nothing.